spec/factories/claim/advocate_claims.rb

```diff
@@ -83,6 +83,7 @@
     end
 
     factory :authorised_claim do
+      offence { FactoryBot.create :offence, :with_fee_scheme, offence_class: FactoryBot.create(:offence_class) }
```

Comment:
does this require the `FactoryBot` prefix? have been trying to remove these unless namespacing is causing an issue

Reply:
**[not a blocker]** As discussed offline, consider using the following instead (not actually tested 😅 ):

```ruby
factory :authorised_claim do
  association :offence, factory: :offence, :with_fee_scheme, :with_offence_class
  ...
end
```

where `with_offence_class` would be a _trait_ on the **offence** factory itself.

Reply:
I've removed the FactoryBot calls.  @lostie Any objection to me creating an Issue for the `association` changes? I don't want to delay this further! :)